tasks/lint_rules/src/eslint-rules.cjs

```diff
@@ -242,6 +285,9 @@ exports.ALL_TARGET_PLUGINS = new Map([
   ],
   ["react-perf", { npm: ["eslint-plugin-react-perf"], issueNo: 2041 }],
   ["nextjs", { npm: ["@next/eslint-plugin-next"], issueNo: 1929 }],
+  ["promise", { npm: ["eslint-plugin-promise"], issueNo: 9999 }], // TODO!
+  ["vitest", { npm: ["eslint-plugin-vitest"], issueNo: 9999 }], // TODO!
+  ["tree-shaking", { npm: ["eslint-plugin-tree-shaking"], issueNo: 9999 }], // TODO!
```

Comment:
tree-shaking has only one rule in node and in oxc. 

You should know better if an Tracking-Issue should be created or not ;)